var/spack/repos/builtin/packages/bazel/package.py

```diff
@@ -187,6 +188,10 @@ def setup_build_environment(self, env):
                 ' --subcommands=pretty_print'
                 ' --jobs={0}'.format(make_jobs))
 
+        # use mkdtemp for temporary directory
+        tmp_path = tempfile.mkdtemp(dir='/tmp', prefix='spack')
+        env.set('TEST_TMPDIR', tmp_path)
```

Comment:
What does this environment variable do? If it's only needed below, there's no reason to create an environment variable.